src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/5.sql

```diff
@@ -0,0 +1,2803 @@
+***************************************************************************/
+IF EXISTS (
+    SELECT *
+    FROM sys.tables
+    WHERE name = 'Instance')
+BEGIN
+    ROLLBACK TRANSACTION
+    RETURN
+END
+
+/*************************************************************
+    Partition Table
+    Table containing Partition Id
+**************************************************************/
+CREATE TABLE dbo.Partition (
+    PartitionKey            BIGINT                 NOT NULL, --PK
+    PartitionId             VARCHAR(64)            NOT NULL    DEFAULT 'Microsoft.Default',
+    --audit columns
+    CreatedDate             DATETIME2(7)           NOT NULL
+) WITH (DATA_COMPRESSION = PAGE)
+
+CREATE UNIQUE CLUSTERED INDEX IXC_Partition ON dbo.Partition
+(
+    PartitionKey
+)
+
+CREATE UNIQUE NONCLUSTERED INDEX IXC_Partition_PartitionKey_PartitionId on dbo.Partition
+(
+    PartitionKey,
+    PartitionId
+)
+
+IF NOT EXISTS (SELECT 1 FROM dbo.Partition WHERE PartitionId = 'Microsoft.Default')
+BEGIN
+    -- Inserting Default values since we are adding PartitionKey as NOT NULL columns in all the tables
+    INSERT INTO dbo.Partition (PartitionKey, PartitionId, CreatedDate)
+    VALUES (1, 'Microsoft.Default', SYSUTCDATETIME())
+END
+
+/*************************************************************
+    Instance Table
+    Dicom instances with unique Partition key, Study, Series and Instance Uid 
+**************************************************************/
+CREATE TABLE dbo.Instance (
+    InstanceKey             BIGINT                     NOT NULL, --PK
+    SeriesKey               BIGINT                     NOT NULL, --FK
+    -- StudyKey needed to join directly from Study table to find a instance
+    StudyKey                BIGINT                     NOT NULL, --FK
+    --instance keys used in WADO
+    StudyInstanceUid        VARCHAR(64)                NOT NULL,
+    SeriesInstanceUid       VARCHAR(64)                NOT NULL,
+    SopInstanceUid          VARCHAR(64)                NOT NULL,
+    --data consistency columns
+    Watermark               BIGINT                     NOT NULL,
+    Status                  TINYINT                    NOT NULL,
+    LastStatusUpdatedDate   DATETIME2(7)               NOT NULL,
+    --audit columns
+    CreatedDate             DATETIME2(7)               NOT NULL,
+    PartitionKey            BIGINT                     NOT NULL DEFAULT 1 --FK
+) WITH (DATA_COMPRESSION = PAGE)
+
+CREATE UNIQUE CLUSTERED INDEX IXC_Instance on dbo.Instance
+(
+    SeriesKey,
+    InstanceKey
+)
+
+--Filter indexes
+CREATE UNIQUE NONCLUSTERED INDEX IX_Instance_PartitionKey_StudyInstanceUid_SeriesInstanceUid_SopInstanceUid on dbo.Instance
+(
+    PartitionKey,
+    StudyInstanceUid,
+    SeriesInstanceUid,
+    SopInstanceUid
+)
+INCLUDE
+(
+    Status,
+    Watermark
+)
+WITH (DATA_COMPRESSION = PAGE)
+
+CREATE NONCLUSTERED INDEX IX_Instance_StudyInstanceUid_Status on dbo.Instance
```

Comment:
Why does not this index have partitionKey?

Reply:
For the Instance table, the new version of the schema will not add any reference to partition. This is because the table already has a clustered index on `StudyKey`, which is partition-specific.